objects/Editors/contained/861b95/contained/48c3ba/object.json

```diff
@@ -532,6 +472,66 @@
       "LuaScript": "elements=\"00000001\"\n",
       "LuaScriptState": "",
       "XmlUI": ""
+    },
+    "1": {
```

Comment:
lets leave sun element at the front of the list

Reply:
I didn't manually put it at the end, TTS did, I think, and is still works the same so does it matter?